cristin-project/src/test/java/no/unit/nva/cristin/projects/CristinApiClientTest.java

```diff
@@ -41,9 +41,9 @@
 public class CristinApiClientTest {
 
     private static final String QUERY_CRISTIN_PROJECTS_EXAMPLE_URI =
-        "https://api.cristin.no/v2/projects/?lang=nb&page=1&per_page=5&title=reindeer";
+        "https://api.cristin-test.uio.no/v2/projects?per_page=5&page=1&lang=nb&title=reindeer";
     private static final String CRISTIN_API_GRANT_ID_SEARCH_EXAMPLE_URI =
-        "https://api.cristin.no/v2/projects/?lang=nb&page=1&per_page=5&project_code=1234567";
+        "https://api.cristin-test.uio.no/v2/projects?per_page=5&project_code=1234567&page=1&lang=nb";
```

Comment:
Will this param ordering always be correct?

Reply:
It stays the same in a deterministic way - if it is in correct logical order is another question...  Maybe we should have 
 something in nva.testutils to decompose an compare URIs